src/style.css

```diff
@@ -0,0 +1,138 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif ;
+}
+
+.productcard {
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgba(243, 243, 243, 1);
+  box-sizing: border-box;
+}
+
+.images_item {
+  width: 160px;
+  height: 134px;
+  margin: 0 3px;
+}
+
+.description {
+  display: flex;
+  flex-direction: column;
+}
+
+.description__name {
+  margin-top: 36px;
+  font-family: "Roboto", sans-serif ;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgba(6, 11, 53, 1);
```

Comment:
why do you use `rgba` if you don't have an alpha-channel (opacity)? Just use `hex`
Apply for other cases